src/st/entity_stage_name_popup_jp.h

```diff
@@ -23,7 +23,7 @@ void EntityStageNamePopup(Entity* self) {
     s16 var_s3;
     s16 var_s2;
 
-#if defined(STAGE_IS_LIB) || defined(STAGE_IS_DAI)
+#if defined(STAGE_IS_LIB)
```

Comment:
@sozud This line conditional applies to dai as well.  Since @Xeeynamo said we're moving away from ovl based ifdefs in favor of multiple versions, that is why I moved to code directly to dai/e_stage_name.c